omod/src/main/java/org/openmrs/module/attachments/fragment/controller/DashboardWidgetFragmentController.java

```diff
@@ -19,6 +20,15 @@
 	
 	public void controller(FragmentModel model, @FragmentParam("patient") PatientDomainWrapper patientWrapper, UiUtils ui,
 	        @InjectBeans AttachmentsContext context) {
+		
+		// Ensure name is HTML-safe to prevent XSS in info messages
+		PersonName htmlSafePersonName = new PersonName(ui.escapeHtml(patientWrapper.getPatient().getGivenName()),
+		        ui.escapeHtml(patientWrapper.getPatient().getMiddleName()),
+		        ui.escapeHtml(patientWrapper.getPatient().getFamilyName()));
+		
```

Comment:
thanks @CrooksDC  can you also remove this space

Reply:
let me try running your changes in on a running openmrs instance

Reply:
Sorry, I removed the spaces @HerbertYiga